File: app/src/main/java/com/erebor/tomkins/pos/view/custom/CommonBindingUtils.java
package com.erebor.tomkins.pos.view.custom;

import android.view.View;
import android.view.ViewGroup;

import androidx.databinding.BindingAdapter;

public class CommonBindingUtils {

    @BindingAdapter("android:layout_width")
    public static void setlayoutWidth(View view, float size) {
        ViewGroup.LayoutParams layoutParams = view.getLayoutParams();
        layoutParams.width = (int) size;
        view.setLayoutParams(layoutParams);
    }
    @BindingAdapter("android:layout_height")
    public static void setlayoutHeight(View view, float size) {
        ViewGroup.LayoutParams layoutParams = view.getLayoutParams();
        layoutParams.height = (int) size;
        view.setLayoutParams(layoutParams);
    }

    @BindingAdapter("android:layout_width")
    public static void setLayoutWidth(View view, int width) {
        ViewGroup.LayoutParams layoutParams = view.getLayoutParams();
        layoutParams.width = width;
        view.setLayoutParams(layoutParams);
    }

    @BindingAdapter("android:layout_height")
    public static void setlayoutHeight(View view, int size) {
        ViewGroup.LayoutParams layoutParams = view.getLayoutParams();
        layoutParams.height = size;
        view.setLayoutParams(layoutParams);
    }

}

File: app/src/main/java/com/erebor/tomkins/pos/view/custom/CommonBindingUtils.kt
package com.erebor.tomkins.pos.view.custom

import android.view.View
import android.view.ViewGroup

import androidx.databinding.BindingAdapter

object CommonBindingUtils {

    @BindingAdapter("android:layout_width")
    fun setlayoutWidth(view: View, size: Float) {
        val layoutParams = view.layoutParams
        layoutParams.width = size.toInt()
        view.layoutParams = layoutParams
    }

    @BindingAdapter("android:layout_height")
    fun setlayoutHeight(view: View, size: Float) {
        val layoutParams = view.layoutParams
        layoutParams.height = size.toInt()
        view.layoutParams = layoutParams
    }

    @BindingAdapter("android:layout_width")
    fun setLayoutWidth(view: View, width: Int) {
        val layoutParams = view.layoutParams
        layoutParams.width = width
        view.layoutParams = layoutParams
    }

    @BindingAdapter("android:layout_height")
    fun setlayoutHeight(view: View, size: Int) {
        val layoutParams = view.layoutParams
        layoutParams.height = size
        view.layoutParams = layoutParams
    }

}
